PoseRPY[] poses
float64 eef_step
